### ISSUE TITLE
Missing Display template for translations.

### ISSUE THREAD

**Issue by preetamslot** [dimitrov-adrian/directus-extension-tags-m2m-interface#3]
Hi, 
This is not a bug but more a question,
is it possible to make use of the display template instead of a string for the tag display?

I have translations in my tags and it would be nice to show the translated title instead of the slug.

Thanks

**Comment by dimitrov-adrian** [comment]
Right now it's not possible. Originally I was thinking of using templates, and at the time of doing the Directus PR, I went with this approach, but after that I realised how ease is to make confusing behaviour, since the search and the original tagged word could differ from those in the template and what is actually selected. I think the same applies to the translations as well.

**Comment by tonyellow** [comment]
I'm interested in this also.

**Comment by dbargaud** [comment]
Yes, this would be a great feature.
Right now, it's unfortunately unusable for the ones like us who uses multi language tagging.

Anyway. thank you for your great work.

**Comment by Xinnerpeace** [comment]
Yes, a great extension but unfortunately no support for translations.
